.Rbuildignore

```diff
@@ -10,3 +10,4 @@
 ^CODE_OF_CONDUCT\.md$
 ^doc$
 ^Meta$
+^.lintr$
```

Comment:
Missing newline at end

Reply:
I didn't see this locally or on GHA. If it still get's flagged on your end please let me know.

Reply:
It was flagged in the diff display, is minor, please ignore